frontstage/models.py

```diff
@@ -57,30 +63,30 @@ def validate_phone_number(form, field):
             input_number = phonenumbers.parse(field.data, 'GB')  # Tell the parser we are looking for a GB number
 
             if not phonenumbers.is_possible_number(input_number):
-                raise ValidationError('This should be a valid phone number between 9 and 15 digits')
+                raise ValidationError(_('This should be a valid phone number between 9 and 15 digits'))
 
             if not phonenumbers.is_valid_number(input_number):
-                raise ValidationError('Please use a valid UK number e.g. 01632 496 0018.')
+                raise ValidationError(_('Please use a valid UK number e.g. 01632 496 0018.'))
```

Comment:
I thought all "please"'s where being removed?

Reply:
Allowing non techs to do this is exactly what this ticket achieves - We haven't been given any chosen strings yet.

Reply:
Ah perfect 👍